bundles/org.openhab.binding.nikobus/README.md

```diff
@@ -196,140 +172,6 @@ Thing push-button 28092A [ impactedModules = "4C6C-2" ]
 
 In addition to the status requests triggered by button presses, there is also a scheduled status update interval defined by the `refreshInterval` parameter and explained above.
 
-## Migration from V1 binding
```

Comment:
@crnjan, this does not belong in the docs, we should not have references to v1 at all. It would be great, though, if you could post this in the [tutorial](https://community.openhab.org/c/tutorials-examples/solutions) section in our forum and we could link to it in the release notes then.

Reply:
I was in doubt about this one indeed